src/main/java/com/kodilla/ecommercee/domain/orders/Item.java

```diff
@@ -29,11 +30,11 @@
     @Column(name = "product_description")
     private String productDescription;
     @Column(name = "price")
-    private Double price;
+    private BigDecimal price;
     @Column(name = "quantity")
-    private Double quantity;
+    private int quantity;
```

Comment:
Proponuję zostawić double, co jeśli ktoś chce kupić 1,5 kg mąki?

Reply:
To kupuje np 2 opakowania. Zostawiłbym int.

Reply:
Nie spotkałem się ze stanami magazynowimi w `double` ;)

Reply:
Może i racja.